loop/routes/home.js

```diff
@@ -45,6 +45,16 @@ module.exports = function(app, conf, logError, storage, tokBox) {
             request.put(req.query.SP_LOCATION, function(error, response) {
               pushStatus = (!error && response.statusCode === 200);
               returnStatus(storageStatus, requestError, pushStatus);
+              if (statsdClient !== undefined) {
+                statsdClient.count('loop.simplepush.call', 1);
+                var counter_push_status_counter;
+                if (pushStatus) {
+                  counter_push_status_counter = 'loop.simplepush.call.heartbeat.success';
+                } else {
+                  counter_push_status_counter = 'loop.simplepush.call.heartbeat.failures';
```

Comment:
I have added the log here: https://github.com/mozilla-services/loop-server/pull/342/files#diff-3c8b9f512b8149de7980ce5ce9930be2R46

Do you want to log a Sentry error?